content/blog/2019-announcing-madstreetden-gold-sponsor.md

```diff
@@ -0,0 +1,18 @@
+Title: Announcing Mad Street Den as Gold Sponsor
+Date: 2019-08-09 09:00
+Category: 2019
+
+We are happy to introduce our Gold sponsor **Mad Street Den**!
+
+<!-- PELICAN_END_SUMMARY -->
+<br>
+<div class="text-center">
+  <a href="https://www.madstreetden.com/" target="_blank">
+    <img src="{filename}/images/sponsors/madstreetden.png" alt="Mad Street Den">
+  </a>
+</div>
+<br>
+
+Mad Street Den is a Computer Vision and Artificial Intelligence Company building the AI architecture of the future. Founded by a Neuroscientist - Designer duo, the company's mission is to build models of generalizable intelligence on scale, that can be deployed through meaningful applications across industries to billions of people across the globe. In 2016, Mad Street Den launched it's flagship brand - Vue.ai. Vue.ai is an end-to-end retail automation platform that is redesigning the future of fashion retail with Artificial Intelligence. Using Image Recognition and Data Science - we extract catalog data, analyze it with user behavior and help your marketing, product and cataloging teams get actionable insights that improve customer experiences, drive conversions and reduce costs. The company recently won the 2019 Venture Beat AI Innovation Awards in the Computer Vision Category, in San Francisco, USA. programming language, became an official language for Android development in 2017.
```

Comment:
There seems to be artifact from a previous text: "programming language, became an official language for Android development in 2017."